README.md

```diff
@@ -27,33 +27,68 @@ See the next section about to get started, the package documentation, the
 
 ### Getting Started
 
+#### Installing drat
+
 You can install the package from [CRAN](http://cran.r-project.org) via the
-standard `install.packages("drat")`.  Alternatively, for a first
-installation, you can also do
+standard `install.packages("drat")`. See below for a way to install drat 
+from it's own drat repository.
 
-```{.r}
-install.packages("drat", repos="http://eddelbuettel.github.io/drat")
-``` 
+#### Adding packages to a new drat repository
 
-to install directly from the drat repository.  After either initial
-installation, you can add line such as
+The following assumes that you want to add a drat repository to your github pages. 
+The github pages are checked out at `/src/repo` and available on the web 
+at <github_username>.github.io.
+
+As a first step, add a `drat` subdirectory (`/src/repo/drat`). This is where 
+new packages should be added to.
+
+Package insertion into a repository then works by passing the package tarball
+filename as an argument to `insertPackage()` or its shorthand variant:
 
```

Comment:
Try http://janschulz.github.io/drat/ and you will get a 404.

I am tired of arguing with you.

Please do not reply. 

Please do not send further emails.